pkg/sdk/datatypes/table.go

```diff
@@ -1,9 +1,15 @@
 package datatypes
 
-// TableDataType is based on TODO [SNOW-1348103]
-// It does not have synonyms.
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/collections"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/logging"
+)
+
+// TableDataType does not have synonyms.
```

Comment:
Let's add at least this link: https://docs.snowflake.com/en/developer-guide/stored-procedure/stored-procedures-java#returning-tabular-data

Reply:
Done